js/content/store.js

```diff
@@ -1385,10 +1385,24 @@ class AppPageClass extends StorePageClass {
         if (this.isDlc()) { return; }
 
         let appid = this.appid;
-        let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
-        if (!response || !response[appid] || !response[appid].success) { return; }
+        let cache = LocalStorage.get("support_infos", {});
+        for (let id in cache) {
+            if (!cache[id].expiry || cache[id].expiry < Date.now()) {
+                delete cache[id];
+            }
+        }
+
+        let supportInfo = cache[appid];
+        if (!supportInfo) {
+            let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
+            if (!response || !response[appid] || !response[appid].success) { return; }
+
+            supportInfo = response[appid].data.support_info;
+            supportInfo.expiry = Date.now() + 2592000000; // 30 days
```

Comment:
```suggestion
            supportInfo.expiry = Date.now() + 30 * 24 * 60 * 60;
```

Reply:
You rather perform a calculation every time? Even though it's neglectable, it seems a little worse for performance.

Reply:
I don't think that this calculation is performed every time, since it's static

Reply:
you forgot `* 1000`

Reply:
I was used to seconds being the smallest unit, since this is what we do with `CacheStorage`

Reply:
Well then you need to change the other `Date.now()` line too.